src/components/UpgradeTableOfContents.tsx

```diff
@@ -128,7 +118,7 @@ const UpgradeTableOfContents: React.FC<IProps> = ({ items, maxDepth = 1 }) => {
         styleType="none"
       >
         <ItemsList items={items} depth={0} maxDepth={maxDepth} />
```

Comment:
Based on the content being supplied, it appears that `depth` and `maxDepth` are unnecessary.

Are there any pages that would create nesting in the ToC?